README.md

```diff
@@ -732,6 +738,15 @@ agenda.on('fail:send email', function(err, job) {
 
 ## Frequently Asked Questions
 
+### What is the order in which jobs run?
+
+Jobs are run **with priority in a first in first out order** (so they will be run in the order they were queued/created _AND_ with respect to highest priority).  We utilize MongoDB's ObjectID through the job record property of `_id` as a sorting value (since it is a [timestamp](https://docs.mongodb.com/manual/reference/method/ObjectId.getTimestamp/) in itself).
```

Comment:
@niftylettuce This is conflicting text, since we are using nextRunAt and not _id.

Suggested text : 
Jobs are run **with priority in a first scheduled first out order** (so they will be run in the order they were scheduled _AND_ with respect to highest priority)